Apps/WebClient/src/Server/Models/OfflineModeConfiguration.cs

```diff
@@ -44,6 +44,6 @@ public class OfflineModeConfiguration
         /// <summary>
         /// Gets or sets the list of IPs that can use the webclient during an outage.
         /// </summary>
-        public IEnumerable<string> Whitelist { get; set; } = Array.Empty<string>();
+        public IList<string> Whitelist { get; set; } = new List<string>();
```

Comment:
Why did you change this?  We don’t really need anything more than an enumerator snd the initialization is cheaper the old way.

Reply:
The reason I changed it is that it wasn't serializing the configuration correctly. Making it a less abstract collection worked better